packages/api/src/util.ts

```diff
@@ -268,6 +274,9 @@ export function getEnv(): BackendEnv {
   const intercom = {
     token: parse('INTERCOM_TOKEN'),
     secretKey: parse('INTERCOM_SECRET_KEY'),
+    webSecret: parse('INTERCOM_WEB_SECRET'),
```

Comment:
Are these ones mandatory by default?

Reply:
No

Reply:
Should we make them mandatory by default? @jacksonh

Reply:
@sywhb no, i think they should be optional so it doesn't break for self hosting. I just wanted to confirm.